responsecode.go

```diff
@@ -77,3 +77,47 @@ var ResponseCodeToName = map[ResponseStatusCode]string{
 	RequestFailedContentNotFound: "RequestFailedContentNotFound",
 	RequestCancelled:             "RequestCancelled",
```

Comment:
while you're refactoring, you could automate this table and String method with https://pkg.go.dev/golang.org/x/tools/cmd/stringer

Reply:
omg where was this tool all my life! You should check over at go-fil-markets and go-data-transfer for some pretty awesome comedy here.